apps/src/templates/sectionProgress/ProgressBox.jsx

```diff
@@ -14,11 +14,25 @@ export default class ProgressBox extends Component {
     style: PropTypes.object,
     lessonIsAllAssessment: PropTypes.bool,
     lessonNumber: PropTypes.number,
+    viewed: PropTypes.bool,
   };
 
   render() {
-    const {started, incomplete, imperfect, perfect, lessonIsAllAssessment} =
-      this.props;
+    const {
+      started,
+      incomplete,
+      imperfect,
+      perfect,
+      lessonIsAllAssessment,
+      viewed,
+    } = this.props;
+
+    const viewedStyle = {
```

Comment:
This adds to the exisiting componet used in the "legend" for the "V1" view.  I added a grey box to this for when work has been viewed.

<img width="585" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/24883357/9197ac76-5c97-4c9e-b443-15d1de98f505">

Reply:
Could you also add a before and after picture?